.github/workflows/canary-gpu-cuda-112.yml

```diff
@@ -38,15 +38,6 @@ jobs:
         with:
           java-version: 11
           distribution: corretto
-      - name: Test MXNet
-        working-directory: canary
-        run: |
-          DJL_ENGINE=mxnet-native-auto ./gradlew clean run
-          rm -rf ~/.djl.ai
-          DJL_ENGINE=mxnet-native-mkl ./gradlew clean run
-          rm -rf ~/.djl.ai
-          DJL_ENGINE=mxnet-native-cu112mkl ./gradlew clean run
```

Comment:
Why we remove them?
The test should be very light-weighted. And this is only place to test mxnet cu112

Reply:
I'm a bit confused on cuda 11.2 for mxnet - it seems like we publish a library to s3 for 11.2 support in S3, and have it documented as well in our code base https://github.com/deepjavalibrary/djl/blob/master/engines/mxnet/mxnet-engine/README.md#linux-gpu

If we don't support 11.2 for mxnet, then the current experience for gpu build is confusing because we don't indicate unsupported architecture, we encounter an missing dependent library exception

Reply:
We support cu112, something is wrong:

```
Suppressed: java.lang.UnsatisfiedLinkError: libopenblas.so.0: cannot open shared object file: No such file or directory
```

Reply:
I agree - seems like we're not downloading all the necessary files. From https://github.com/deepjavalibrary/djl-demo/runs/7794346123?check_suite_focus=true I see that we're only downloading one of the necessary .so files

```
[DEBUG] - Using cache dir: /root/.djl.ai/mxnet/1.9.0-20220318-cu112mkl-linux-x86_64
[INFO ] - Extracting native/lib/libmxnet.so to cache ...
Error: Exception in thread "main" ai.djl.engine.EngineException: Failed to load MXNet native library
[DEBUG] - Loading mxnet library from: /root/.djl.ai/mxnet/1.9.0-20220318-cu112mkl-linux-x86_64/libmxnet.so
```

We're not downloading any of the common .so files from s3